service/storage_service.cc

```diff
@@ -462,6 +466,9 @@ future<storage_service::nodes_to_notify_after_sync> storage_service::sync_raft_t
 
         if (const auto ip = am.find(id)) {
             co_await remove_ip(*ip, host_id, true);
+            utils::get_local_injector().inject("notify_node_removal", [purged_ip=*ip] {
+                        slogger.info("Purged node:{} from gossiper", purged_ip);
```

Comment:
Using error injection just to print a log message looks like total overkill

Doesn't `gossiper::force_remove_endpoint` (called from `remove_ip`) already provide a message we could rely on?

Alternatively, we could print a message here unconditionally (not inside error injection). To avoid polluting Scylla log in production, it could be done on DEBUG level instead of INFO, and you could enable DEBUG level for `slogger` in the test

Reply:
Done, removed the new injection